src/main/java/no/entur/uttu/stopplace/filter/StopPlaceFilter.java

```diff
@@ -1,3 +1,13 @@
 package no.entur.uttu.stopplace.filter;
 
-public interface StopPlaceFilter {}
+public interface StopPlaceFilter {
```

Comment:
This interface is probably misnamed. It is not really a filter, but rather a configuration or parameter wrapper for a filter. So I suggest renaming it to StopPlaceFilterParameters. Likewise, the implementations should be renamed to follow the same pattern, e.g. QuayIdFilter -> QuayIdFilterParameter. 

This leaves the name StopPlaceFilter for the class actually doing the filtering, now somewhat awkwardly named "StopPlaceFilterer".